src/telemetry/telemetry.ts

```diff
@@ -119,9 +120,19 @@ class Telemetry implements ITelemetry {
     telemetry.trackEvent(WINDOW_OPEN_EVENT, properties);
   }
 
-  public trackDeviceCharacteristics(properties?: any){
-    properties = properties || {}
+  public trackDeviceCharacteristicsTelemetry(properties?: any){
+    const deviceProperties = {
+      deviceHeight: screen.height,
+      deviceWidth: screen.width,
+      browserScreenSize: getBrowserScreenSize(window.innerWidth),
+      browserHeight: window.innerHeight,
+      browserWidth: window.innerWidth,
+      scale: getDeviceScreenScale()
+    };
+    properties = properties || deviceProperties;
```

Comment:
This should probably be the union of the two objects:
```suggestion
    properties = Object.assign(properties || {}, deviceProperties);
```
This will cause any property on properties that is not on deviceProperties to be preserved while using the value from deviceProperties for any common properties. cf: https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/object/assign